.github/workflows/release-workflow.yml

```diff
@@ -29,15 +28,19 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew build buildDeb buildRpm --no-daemon -Dbuild.snapshot=false
 
+      # Creating artifact path as well as individual folders for rpm, zip and deb assets
       - name: Create Artifact Path
         run: |
           mkdir -p alerting-artifacts
```

Comment:
Looks like you create `alerting-artifacts` directory before trying to copy the artifacts into it, whereas you're directly copying the artifacts into the new directories before creating them, will this cause any issues?

Reply:
For each build, I assumed there would be one each zip, rpm and deb files. But if there is a possibility for more then I need to create directories for each one before copying them. Right now it is basically renaming the zip, rpm and deb file and using that to attach as asset.